packages/odie-client/package.json

```diff
@@ -39,11 +39,11 @@
 		"react-dom": "^18.2.0",
 		"react-intersection-observer": "^9.4.3",
 		"tslib": "^2.3.0",
-		"typescript": "^5.3.2",
+		"typescript": "^5.3.3",
 		"wpcom-proxy-request": "^7.0.5"
 	},
 	"devDependencies": {
 		"@automattic/calypso-typescript-config": "workspace:^",
-		"typescript": "^4.7.4"
+		"typescript": "^4.9.5"
```

Comment:
I don't see a good reason to keep this dev dependency here, given that we have a newer version in `dependencies`. 

Furthermore `typescript` should be in `devDependencies`, no need to be in `dependencies`. Will correct that before shipping.

Reply:
✅  https://github.com/Automattic/wp-calypso/pull/86683/commits/89d672d291e2ee50bbbd7df97e073710613b3995